x-pack/plugins/security_solution/public/common/components/alerts_viewer/alerts_table.tsx

```diff
@@ -79,6 +80,7 @@ const AlertsTableComponent: React.FC<Props> = ({
   const dispatch = useDispatch();
   const alertsFilter = useMemo(() => [...defaultAlertsFilters, ...pageFilters], [pageFilters]);
   const { filterManager } = useKibana().services.data.query;
+  const ACTION_BUTTON_COUNT = 3;
```

Comment:
💅 Could be moved to outside of `AlertsTableComponent` since it's a constant value.